packages/core/src/utils.ts

```diff
@@ -98,6 +98,18 @@ export function convertBigIntToByteArray(bigInt: bigint) {
   return result.reverse()
 }
 
+export function convertCircuitBigIntToString(input: bigint | string) {
```

Comment:
On second thoughts, we can call it `convertRevealBigIntToString`

Reply:
Sure, I'll change this!